src/components/BContainer/BContainer.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <section>
+    <component :is=headerElement class="header" v-if="label">{{ label }}</component>
```

Comment:
Nice

Reply:
Can we put those attributes on multiple lines?
https://vuejs.org/v2/style-guide/#Multi-attribute-elements-strongly-recommended